pdm/termui.py

```diff
@@ -216,21 +215,18 @@ def logging(self, type_: str = "install") -> Iterator[logging.Logger]:
         file_name = mktemp(".log", f"pdm-{type_}-")
 
         if self.verbosity >= Verbosity.DETAIL:
-            handler: logging.Handler = RichHandler(
-                console=_err_console, show_time=False, show_level=False, show_path=False
-            )
+            handler: logging.Handler = logging.StreamHandler()
             handler.setLevel(LOG_LEVELS[self.verbosity])
         else:
             handler = logging.FileHandler(file_name, encoding="utf-8")
             handler.setLevel(logging.DEBUG)
         handler.setFormatter(logging.Formatter("%(name)s: %(message)s"))
-        logger.handlers[1:] = unearch_logger.handlers[:] = [handler]
+        logger.addHandler(handler)
+        unearch_logger.addHandler(handler)
```

Comment:
sorry for all the typo fixes, but this should be `unearth` :sweat_smile:

Reply:
Thanks, fixed. I dunno why I always mistype that word, yeah, not the first time.